src/components/LoadingScreen/AnimatedLogo.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { keyframes } from 'styled-components'
+import { theme } from '@aragon/ui'
+
+const PATH = `
+  M75 19a161.38 161.38 0 0 0-7.37 2.68 214.8 214.8 0 0 0-14.98 6.5
+  C38.81 34.8 25.92 42.74 15 52
+  c.59 2.94 1.49 6.6 2.78 10.86
+  a136.54 136.54 0 0 0 8.39 21.24
+  c5.17 10.43 11.44 19.48 18.64 26.95
+  A80.72 80.72 0 0 0 75 131
+  a80.86 80.86 0 0 0 30.18-19.98
+  c7.18-7.44 13.46-16.53 18.65-27
+  a138.28 138.28 0 0 0 8.38-21.2
+  c1.29-4.24 2.2-7.9 2.79-10.82-10.92-9.25-23.81-17.18-37.67-23.82
+  a215.66 215.66 0 0 0-14.99-6.5
+  A161.55 161.55 0 0 0 75 19z
+`
+
+const STROKE_WIDTH = 3
+const LOOP_DURATION = 1800
+const DONE_TRANSITION_DURATION = 400
+const DASH_ARRAY = 348.5 // circumference, found by manually incrementing it :o
```

Comment:
https://stackoverflow.com/questions/44006664/the-calculation-behind-progress-circle-using-dasharray-and-dashoffset lists some other values, maybe not needed but might help to round up even more.

Reply:
Do you mean, to calculate the circumference? The problem here is that the shape is not a circle − otherwise I would have reused the same calculation than for [`CircleGraph`](https://github.com/aragon/aragon-ui/blob/master/src/components/CircleGraph/CircleGraph.js#L13).

Reply:
Ahhh, I was mislead by the comment, thanks for clarifying